### ISSUE TITLE
All the images are broken?

### ISSUE THREAD

**Issue by ghost** [githubflow/githubflow.github.io#4]
I'm looking at http://githubflow.github.io/ , and the images (which have URLs like "https://img.skitch.com/20110831-jcmx5qktw5beabq7q1rcpj1k6c.png") are broken.  Going to them directly, I get redirected to https://evernote.com/skitch/ .  Can they be fixed?

**Comment by garster** [comment]
Wow this is till open 5 years later. The content is good but I would love the images. Guess it's a dead repo.

**Comment by pndlsm** [comment]
Yup, still broken. Which is a real pity because its probably the best explanation I've found around otherwise.

**Comment by toddprouty** [comment]
@schacon The pull request above fixes this issue. It's a great article and still relevant, might as well restore the images!

**Comment by devinschumacher** [comment]
images still broken
